CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unrelease] -
 
+### Fixed
+
+- Correction of the task message generated during escalation
+
+### Added
+
+- Configuring transiflex as a CI
+
```

Comment:
? XD

Reply:
You can remove this part, as it doesn't affect the use of the plugin, it's just internal stuff.